.circleci/config.yml

```diff
@@ -1,20 +1,19 @@
 version: 2.1
 orbs:
-  gcp-gcr: circleci/gcp-gcr@0.16.9
+  gcp-gcr: circleci/gcp-gcr@0.16.10
   pulumi: pulumi/pulumi@2.1.0
-  gcp-cli: circleci/gcp-cli@3.3.0
+  gcp-cli: circleci/gcp-cli@3.3.1
 jobs:
   build:
     docker:
-      - image: cimg/node:22.13
+      - image: cimg/node:22.15
```

Comment:
Keep it the same until we update main code? I know there are not really breaking changes, just easier to update in one go with search and replace.